Fulfillment/Integration/01_ManageConnections.md

```diff
@@ -0,0 +1,143 @@
+
+Edit a connection to adjust any changed credentials, the connection name or further settings. The driver selection is read-only and cannot be edited.
+
+#### Prerequisites
+
+At least one connection has been created, see [Create a connection](#create-a-connection).
+
+#### Procedure
+
+*Fulfillment > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connections]")
+
+1. Click the connection you want to edit in the list of connections.   
+    The *Edit connection* view is displayed.
+
+    > [Info] Depending on the driver of the connection, the *Credentials* and the *Settings* tab are displayed in the *Edit connection* view.
+
+    ![Edit connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/EditConnectionCredentials.png "[Edit connection]")
+
+2. Edit the desired data of the connection in the corresponding fields in the *Credentials* tab.
+
+3. If necessary, click the *Settings* tab and edit the desired settings of the connection in the corresponding fields.
+
+4. Click the [SAVE] button.   
+    The *Checking credentials...* notice is displayed.
+
+    ![Credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/CheckingCredentials.png "[Credentials]")
+
+    The *Edit connection* view is automatically closed and the *Connections* view is displayed again.
+
+
+## Enable a connection
+
+A connection can only be used if it is active. Otherwise, no data is being synchronized via the connection. By default, all connections that have been established are first inactive and therefore must be enabled. If a connection has been disabled temporarily, it can be enabled it again. It is possible to enable several inactive connections at a time.
+
+#### Prerequisites
+
+- At least one connection has been created, see [Create a connection](#create-a-connection).
+- At least one connection is inactive, see [Disable a connection](#disable-a-connection).
+
+#### Procedure
+
+*Fulfillment > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connections]")
+
+1. Select the checkbox of the connection you want to enable in the list of connections.   
+    The editing toolbar is displayed above the list of connections.
+
+2. Click the [ENABLE] button in the toolbar.   
+    The connection has been activated.
+
+3. If necessary, click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button to update the list of connections.   
+    The status of the enabled connection in the *Status* column has switched to **Active**.
+
+[comment]: <> (Update der Liste läuft automatisch aber es dauert ein wenig. Refresh step vorischtshalber drin lassen?)
+
+
+## Disable a connection
+
+Once a connection has been established, it cannot be deleted. Nevertheless, the connection can be disabled if it is no longer used. It is possible to disable several active connections at a time.
```

Comment:
```suggestion
Once a connection has been established, it cannot be deleted. Nevertheless, the connection can be disabled to prevent any further data from being synchronized via the connection, for example, if it is no longer used. It is possible to disable several active connections at a time.
```